base/BaseSequelize.js

```diff
@@ -0,0 +1,51 @@
+import autoBind from "auto-bind";
+
+
+// thuộc repository pattern
+class BaseSequelize {
```

Comment:
- Just using MongoDB

Reply:
well, i comment it because i think our team want to do hybrid db